src/main/java/opendota/Parse.java

```diff
@@ -306,7 +307,18 @@ public void onChatEvent(Context ctx, CDOTAUserMsg_ChatEvent message) {
         entry.value = value;
         output(entry);
     }
-    
+
+    // New chat event
+    @OnMessage(CDOTAUserMsg_ChatMessage.class)
+    public void onAllChatMessage(Context ctx, CDOTAUserMsg_ChatMessage message) {
+        int type = message.getChannelType();
+        Entry entry = new Entry(time);
+        entry.slot = message.getSourcePlayerId();
```

Comment:
can you verify this is 0-4 and 128-132?

Reply:
0-9 like the old chat

Reply:
that is probably fine as long as it matches the old `        entry.slot = getEntityProperty(e, "m_iPlayerID", null);`